src/components/application_manager/include/application_manager/commands/mobile/send_location_request.h

```diff
@@ -100,6 +100,14 @@ class SendLocationRequest : public CommandRequestImpl {
 
   bool CheckHMICapabilities(
       std::vector<hmi_apis::Common_TextFieldName::eType>& fields_names);
+
+  /**
+   * @brief IsSendLocationEnabled
+   * @return true in case when sendLocationEnabled true in navigationCapability
+   * otherwise return false
+   */
+  bool IsSendLocationEnabled() const;
+
```

Comment:
@LitvinenkoIra 
I think there is no need to put this method into header file. Create free function in .cc instead and pass application_manager_.hmi_capabilities() as parameter

Reply:
@dev-gh Done ae19d3442426a8be0d00b0698358ada960cc16c3